### PR TITLE
[docs] Add cell guide notebook for queries on cell type descendants 

### LINKED ISSUE #1029: Add include_cell_type_descendants convenience parameter to get_anndata()
Add `include_cell_type_descendants` parameter to `get_anndata()`.

**Proposal**

```
adata = cellxgene_census.get_anndata(
    census = census,
    organism = "Homo sapiens",
    feature_ids: ['ENSG00000161798', 'ENSG00000188229'],
    cell_type = ['microglial cell', 'neuron'],
    include_cell_type_descendants = True,
    column_names = {"obs": ["assay", "cell_type", "tissue", "tissue_general", "suspension_type", "disease"]},
)
```

This would use the `get_term_descendants` function in [cellxgene-ontology-guide](https://chanzuckerberg.github.io/cellxgene-ontology-guide/cellxgene_ontology_guide/ontology_parser.html#OntologyParser.get_term_descendants) without requiring the user to use an additional API.

**User Quote**
> I’d like a toolkit to easily navigate descendants and ascendants of cell types and tissues

### PR DESCRIPTION
**Pull request by pablo-gar**
Fixes #1029

### CODE REVIEW

File: api/python/notebooks/api_demo/census_and_cell_guide_example.ipynb [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+    "1. Query Census for all descendant terms of a cell type\n",
+    "\n",
+    "⚠️ Note that the Census RNA data includes duplicate cells present across multiple datasets. Duplicate cells can be filtered in or out using the cell metadata variable `is_primary_data` which is described in the [Census schema](https://github.com/chanzuckerberg/cellxgene-census/blob/main/docs/cellxgene_census_schema.md#repeated-data).\n",
+    "\n",
+    "\n",
+    "## Obtain all descendant terms of a cell type\n",
+    "\n",
+    "The [CELLxGENE Ontology Guide](https://github.com/chanzuckerberg/cellxgene-ontology-guide/) provides a programatic interface to access and and parse the ontologies used by CELLxGENE for its cell and gene metadata. You can perform basic ontological operations, for example obtaining all ascendants or descendants of a cell type according to the Cell Ontology (CL).\n",
+    "\n",
+    "To learn more about all the capabilities of the CELLxGENE Ontology Guide, please visit its [API documentation](https://chanzuckerberg.github.io/cellxgene-ontology-guide/cellxgene_ontology_guide.html)).\n",
```
Comment (ebezzi) [inline api/python/notebooks/api_demo/census_and_cell_guide_example.ipynb:28]:
```suggestion
    "To learn more about all the capabilities of the CELLxGENE Ontology Guide, please visit its [API documentation](https://chanzuckerberg.github.io/cellxgene-ontology-guide/cellxgene_ontology_guide.html).\n",
```

**Review by ebezzi — APPROVED** [review]
One small typo, then LGTM.

**Comment by brianraymor** [comment]
@nayib-jose-gloria @Bento007 - would you take a quick look?

File: api/python/notebooks/api_demo/census_and_cell_guide_example.ipynb [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+   "source": [
+    "schema_version = census_info.iloc[2, 2]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "995b0adc-1a8d-4220-9577-2c38ceba9a3b",
+   "metadata": {},
+   "source": [
+    "Now we can instantiate an `OntologyParser` to access the appropriate ontology versions according the dataset schema.  ma version."
```
Comment (pablo-gar) [inline api/python/notebooks/api_demo/census_and_cell_guide_example.ipynb:160]:
```suggestion
    "Now we can instantiate an `OntologyParser` to access the appropriate ontology versions according the dataset schema version."
```

**Review by Bento007 — APPROVED** [review]

**Pull request merged**